src/styles.css

```
@@ -213,7 +213,7 @@
 .sidebar-list ul li .submenu-items a:active {
   border-style: solid;
 }
-.sidebar-list ul li .submenu-items li.active > a {
+.sidebar-list ul li .submenu-items li.active > div.method-button-wrapper > a {
```

Can we simplify it to just say that any descendant `a` under the `li.active` will be selected?
```suggestion
.sidebar-list ul li .submenu-items li.active a {
```